src/OrchardCore/OrchardCore.Users.Core/Models/UserMenu.cs

```diff
@@ -0,0 +1,5 @@
+namespace OrchardCore.Users.Models;
+
+public class UserMenu
+{
+}
```

Comment:
Yes I understand but looks strange to have an empty model.

No time to look at the details but maybe could hold useful properties.

Maybe at least inherit from `IEntity` like `ISite` so that drivers/views could set/get properties.

Reply:
sure. done